blazor/datagrid/lazy-load-grouping.md

```diff
@@ -215,94 +363,16 @@ The following code example describes the lazy load grouping handled at the serve
-                "OLDWO", "ROMEY", "LONEP", "ANATR", "HUNGO", "THEBI", "DUMON", "WANDK", "QUICK", "RATTC", "ISLAT", "RATTC", "LONEP", "ISLAT", "TORTU",
-                "WARTH", "ISLAT", "PERIC", "KOENE", "SAVEA", "KOENE", "BOLID", "FOLKO", "FURIB", "SPLIR", "LILAS", "BONAP", "MEREP", "WARTH", "VICTE",
-                "HUNGO", "PRINI", "FRANK", "OLDWO", "MEREP", "BONAP", "SIMOB", "FRANK", "LEHMS", "WHITC", "QUICK", "RATTC", "FAMIA" };          
-            string[] Product = { "Chai", "Chang", "Syrup", "Corn Snacks", "Gumbo Mix", "Seeds",
-                "Dried Pears", "Sauce", "Mishi Kobe Niku", "Ikura", "Queso Cabrales", "Queso Manchego Pastora", "Konbu",
-                "Tofu", "Genen Shouyu", "Pavlova", "Alice Mutton", "Biscuits", "Teatime Chocolate Biscuits", "Sir Rodney\"s Marmalade", "Sir Rodney\"s Scones",
-                "Gustaf\"s Knäckebröd", "Tunnbröd", "Guaraná Fantástica", "Nougat-Creme", "Gumbär Gummibärchen", "Schoggi Schokolade", "Rössle Sauerkraut",
-                "Thüringer Rostbratwurst", "Nord-Ost Matjeshering", "Gorgonzola Telino", "Mascarpone Fabioli", "Geitost", "Sasquatch Ale", "Steeleye Stout", "Inlagd Sill",
-                "Gravad lax", "Nuts", "Chips", "Crab Meat", "Jack\"s Clam Chowder", "Singaporean Fried Mee", "Ipoh Coffee",
-                "Gula Malacca", "Rogede sild", "Spegesild", "Zaanse koeken", "Chocolade", "Maxilaku", "Valkoinen suklaa", "Manjimup Dried Apples", "Filo Mix", "Perth Pasties",
-                "Tourtičre", "Pâté chinois", "Ipoh Coffee", "Ravioli Angelo", "Escargots Bourgogne", "Raclette Courdavault", "Cake", "Sirop d\"érable",
-                "Tarte au sucre", "Vegie-spread", "Lakkalikri", "Louisiana Pepper Sauce", "Louisiana Hot Spiced Okra", "Lumberjack Lager", "Scottish Longbreads",
-                "Gudbrandsdalsost", "Outback Lager", "Flotemysost", "Mozzarella di Giovanni", "Röd Kaviar", "Longlife Tofu", "Rhönbräu Klosterbier", "Lakkalikööri", "Original Frankfurter" };
-            int OrderID = 1001;
-            int i = 0; int j = 0; int k = 0; int l = 0; int m = 0;
-            for (int x = 0; x < 500000; x++)
-            {
-                i = i >= CustomerId.Length ? 0 : i; 
-                l = l >= Product.Length ? 0 : l; 
-                customers.Add(new Customer() 
-                { 
-                    OrderID = OrderID + x, 
-                    ProductID = x % 1000,
-                    CustomerID = CustomerId[i],                  
-                    ProductName = Product[l],  
-                    UnitsInStock = new Random().Next(1, 10000) 
-                });
-                i++; j++; k++; l++; m++;
-            }
-            return customers;
-        }
-    }
-}
-```
-
-The following GIF represents the lazy load grouping with infinite scrolling functionality in DataGrid
 
-![Lazy Load grouping with Infinite Scrolling in Blazor DataGrid](./images/blazor-datagrid-lazy-load-infinite-scrolling.gif)
+## Limitations for lazy load grouping
```

Comment:
ensure whether the print is possible or not

Reply:
Modified